tests/_support/Config/Registrar.php

```diff
@@ -83,6 +83,7 @@ class Registrar
             'failover'    => [],
             'port'        => 3306,
             'foreignKeys' => true,
+            'synchronous' => 0,
```

Comment:
Sorry, can we remove this? It's not necessary since the value is in the config file.

Reply:
Ok, I missed that GitHub Actions is affected. Thanks.